src/test/java/fr/esgi/al5_2/Tayarim/ProprietaireServiceTest.java

```diff
@@ -26,262 +26,281 @@
-        doNothing().when(proprietaireRepository).deleteById(id);
-
-        ProprietaireDTO proprietaireDTO = proprietaireService.deleteProprietaire(id);
-
-        Assertions.assertThat(proprietaireDTO).isNotNull();
-        Assertions.assertThat(proprietaireDTO.getId()).isEqualTo(id);
-        Assertions.assertThat(proprietaireDTO.getNom()).isEqualTo(proprietaire.getNom());
-        Assertions.assertThat(proprietaireDTO.getPrenom()).isEqualTo(proprietaire.getPrenom());
-        Assertions.assertThat(proprietaireDTO.getEmail()).isEqualTo(proprietaire.getEmail());
-        Assertions.assertThat(proprietaireDTO.getNumTel()).isEqualTo(proprietaire.getNumTel());
-        Assertions.assertThat(proprietaireDTO.getDateInscription()).isEqualTo(localDateTime);
-        Assertions.assertThat(proprietaireDTO.getLogements()).isNull();
-
-    }
-
-    @Test
-    public void ProprietaireService_VerifyPassword_ReturnTrue(){
-        Long id = 1L;
-        LocalDateTime localDateTime = LocalDateTime.now();
-        String password = "password";
-        String hashedPassword = "$2a$12$3hQDUblvPShmuQg/.g0Qk.wHAGjqPL54RMO/lNgsei/HQGo0ZLIYm";
-
-        Proprietaire proprietaire = Proprietaire.builder()
-                .nom("Ferreira")
-                .prenom("Mathieu")
-                .email("test@gmail.com")
-                .numTel("0612345678")
-                .motDePasse(hashedPassword)
-                .dateInscription(localDateTime)
-                .build();
-        when(proprietaireRepository.findById(id)).thenReturn(Optional.ofNullable(proprietaire));
-
-        boolean result = proprietaireService.verifyPassword(password, id);
-
-        Assertions.assertThat(result).isTrue();
-
-
-    }
-
-    @Test
-    public void ProprietaireService_VerifyPassword_ReturnFalse(){
-        Long id = 1L;
-        LocalDateTime localDateTime = LocalDateTime.now();
-        String password = "passwordFalse";
-        String hashedPassword = "$2a$12$3hQDUblvPShmuQg/.g0Qk.wHAGjqPL54RMO/lNgsei/HQGo0ZLIYm";
-
-        Proprietaire proprietaire = Proprietaire.builder()
-                .nom("Ferreira")
-                .prenom("Mathieu")
-                .email("test@gmail.com")
-                .numTel("0612345678")
-                .motDePasse(hashedPassword)
-                .dateInscription(localDateTime)
-                .build();
-        when(proprietaireRepository.findById(id)).thenReturn(Optional.ofNullable(proprietaire));
-
-        boolean result = proprietaireService.verifyPassword(password, id);
-
-        Assertions.assertThat(result).isFalse();
-
-
-    }
+  @Mock
+  private ProprietaireRepository proprietaireRepository;
+
+  @InjectMocks
+  private ProprietaireService proprietaireService;
+
+  @Test
+  public void ProprietaireService_CreerProprietaire_ReturnsProprietaireDto() {
+
+    LocalDateTime localDateTime = LocalDateTime.now();
+
+    ProprietaireCreationDTO proprietaireCreationDTO = ProprietaireCreationDTO.builder()
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.AbbreviationAsWordInNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Abbreviation in name 'proprietaireCreationDTO' must contain no more than '1' consecutive capital letters.